README.md

```diff
@@ -69,6 +69,16 @@ import SwagGenKit
 import Swagger
 ```
 
+## Docker
+
+`airtasker/swaggen` image acts as a standalone executable. It can be used as an alternative to installing via homebrew, or for developers who are unable to install Xcode.
```

Comment:
Is this supposed to be referencing airtasker?

Reply:
It shouldn't. As a user of the docker image, I will not trust it unless it is published by @yonaskolb , which is why I'm building my [own](https://cloud.docker.com/u/mayachit/repository/docker/mayachit/swaggen)

Reply:
Hi,

I do think that having a first party Docker image is crucial for (CI use for exemple).
So I build did my own Docker image by forking @mithun(**great**) Docker image.

I have made a couple of tweaks from @mithun(**great**) Docker image:
- updating it to the latest version of Swift 
- making it generate image with Swift slim image - **4.2.0**(467.93 MB) -> **4.2.0-slim**(77.05 MB)
- add a bunch of variable to make it easy to generate images for various tag of my fork of **SwagGen**

https://hub.docker.com/r/hawkci/swaggen

Reply:
After a few months of working with Swaggen in a Docker container I do prefer this workflow. Because it guarantees that it is working the same way as our CI is working. With one script anyone in the team can generate a custom version of the library. It's easier to maintain and update. FWIW I do update my Swaggen image for every new Swift build.
Last update was this morning with Swift 5.1.5 it's pretty easy to do.